### PR TITLE
Origin/authzgalla dev billing microsoft.billing 2020 05 01

### PR DESCRIPTION
**Pull request by RitikaGeraMicrosoft**
### ### <i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json [hunk 1]
```diff
@@ -2673,6 +2673,9 @@
         "x-ms-examples": {
           "BillingAccountPermissionsList": {
             "$ref": "./examples/BillingAccountPermissionsList.json"
+          },
+          "BillingAccountEaPermissionsList": {
```
Comment (tugupt) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json:2677]: Why are we having seperate, EA permission list?

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json [hunk 2]
```diff
@@ -3078,10 +3165,13 @@
         "x-ms-examples": {
           "BillingAccountRoleAssignment": {
             "$ref": "./examples/BillingAccountRoleAssignment.json"
+          },
+          "BillingAccountEaRoleAssignment": {
+            "$ref": "./examples/BillingAccountEaRoleAssignment.json"
```
Comment (tugupt) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json:3170]: Here also

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing_master.json [hunk 3]
```diff
@@ -0,0 +1,6399 @@
...
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/listInvoiceSectionsWithCreateSubscriptionPermission": {
```
Comment (tugupt) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing_master.json:170]: this is not EA. why are we publishing this?

File: specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json [hunk 4]
```diff
@@ -3147,6 +3240,52 @@
             }
           }
         }
+      },
+      "put": {
+        "tags": [
+          "BillingRoleAssignments"
+        ],
+        "operationId": "RoleAssignments_Put",
```
Comment (allenjzhang) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json:3248]: BillingRoleAssignments_CreateOrUpdate?

Comment (allenjzhang) [inline specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json:3248]: >"RoleAssignments [](start = 23, length = 16)

BillingRoleAssignments_Put

**Comment by asarkar84** [comment]
This may require a new version as per new Azure breaking change policy: https://aka.ms/azbreakingchangespolicy

**Comment by RitikaGeraMicrosoft** [comment]
Closing until further upates are received.

**Pull request closed without merging**